packages/table/src/Table/Table.spec.tsx

```diff
@@ -206,6 +224,37 @@ describe('packages/table/Table', () => {
     });
   });
 
+  describe('column visibility', () => {
+    test('renders the correct cells in a row by default', () => {
+      render(<TableWithHook hasToggleCheckboxes />);
```

Comment:
I feel like adding these checkboxes to the test components overcomplicates the tests... But the alternative might be just as complex:

```ts
test('', () => {
  // Render the table as a test hook
  const tableResult = renderHook(useLeafyGreenTable);
  const table = tableResult.result.table;

  // render the table
  const renderResult = render(<Table table={table} .../>);

  // Manually trigger the visibility toggle
  const columnToHide = tableResult.result.getColumn('...');
  const columnToggleFn = columnToHide.getToggleVisibilityHandler();  
  act(() => {
    columnToggleFn();
  })

  // rerender the table
  renderResult.rerender(<Table table={table} ... />);

  expect(...) // same expect
})
```

Reply:
Can you elaborate more on "overcomplicates the tests"?